src/app/plugins/code.plugin.ts

```diff
@@ -0,0 +1,48 @@
+import Constants from '../../common/constants';
+import { Plugin } from '../../common/plugin';
+import { ChannelType, IContainer, IMessage } from '../../common/types';
+
+export class CodePlugin extends Plugin {
+  public name: string = 'Code Plugin';
+  public description: string =
+    'Shows someones text in code form (it only works in the channel the command was called in). "Code how" posts a link with formatting help and a quick discord message coding explanation.';
+  public usage: string = 'code <message ID/link (optional)> <lang (optional)>/ code how';
+  public pluginAlias = [''];
+  public permission: ChannelType = ChannelType.Public;
+
+  constructor(public container: IContainer) {
+    super();
+  }
+
+  public async execute(message: IMessage, args?: string[]) {
+    const input = args || ['', ''];
+
+    if (input[0] === '' || input[0] === 'how') {
+      message.channel.send(
+        `You can post in Discord like a real boss by refering to this guide ${Constants.DiscordFormatingInfo}.  The coding in discord information is located under the third header.\n` +
+          `>>> How to code in discord, speed course edition: \n` +
+          `\'''<language exstension(optional)>` +
+          `\n<Code>\n` +
+          `\'''\n` +
+          `Replace the ' with \``
+      );
```

Comment:
Not sure if this breaks any good practices, but I think it makes this block a lot more readable.  If you could let me know if this is a nono.